src/storage-client/src/healthcheck.rs

```diff
@@ -83,7 +83,10 @@ pub static MZ_SESSION_HISTORY_DESC: Lazy<RelationDesc> = Lazy::new(|| {
             "connected_at",
             ScalarType::TimestampTz { precision: None }.nullable(false),
         )
-        .with_column("application_name", ScalarType::String.nullable(false))
```

Comment:
Does any user stuff depend on this old name? Is it ok to change it?

Reply:
Nope! Only one customer has even heard of this table, and they have explicitly been told that columns/tables are subject to change.